src/modules/auth/auth.service.ts

```diff
@@ -74,4 +78,67 @@ export default class AuthenticationService {
       );
     }
   }
+
+  async generateRefreshToken(user: any) {
+    return this.jwtService.sign(
+      {
+        email: user.email,
+        first_name: user.first_name,
+        last_name: user.last_name,
+        sub: user.id,
+      },
+      {
+        secret: appConfig().jwtRefreshSecret,
+        expiresIn: appConfig().jwtRefreshExpiry,
+      }
+    );
+  }
+
+  async generateAccessToken(user: any) {
+    return this.jwtService.sign(
```

Comment:
Well done on the implementation. the only issue i see is the storage of user personal information in the jwt token, its not best practice security wise. its better to have only the user id in it.